react/react1/week2/todo-app/src/components/TodoList.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from "react";
+import TodoItem from "./TodoItems";
+import AddRandomTodo from "./AddRandomTodo";
+
+function ToDosList({ todoItem }) {
+  const [todos, setTodo] = useState(todoItem);
+
+  const mapTodo = () => {
+    if (todos.length !== 0) {
+      return todos.map((item) => (
+        <TodoItem key={item.id} item={item} todos={todos} setTodo={setTodo} />
+      ));
+    } else {
+      return <h3>No items</h3>;
+    }
+  };
```

Comment:
Nice approach to move this into a function. I would suggest to change the naming to a more typical "renderX" though. 
It makes sense that you are mapping the items and that is what you call it, but to somebody else working with the code it is not very informative from the first glance - mapping could be adding or recalculating properties, or something else. 
``renderTodos`` would clearly indicate what this is 👍🏽